src/parsing/parser.rs

```diff
@@ -7941,11 +7941,18 @@ fn jsx_space_separator(previous_node: &Node, current_node: &Node, context: &Cont
     }
   }
 
+  fn get_quote_char(context: &Context) -> String {
+    return match context.config.quote_style {
+      QuoteStyle::PreferDouble | QuoteStyle::AlwaysDouble => "\"".to_string(),
+      QuoteStyle::PreferSingle | QuoteStyle::AlwaysSingle => "\'".to_string()
+    };
+  }
+
   fn jsx_force_space_with_newline_if_either_node_multi_line(previous_node: &Node, current_node: &Node, context: &Context) -> PrintItems {
     let previous_node_info_range = get_node_info_range(previous_node, context);
     let current_node_info_range = get_node_info_range(current_node, context);
     let spaces_between_count = node_helpers::count_spaces_between_jsx_children(previous_node, current_node, &context.module);
-    let jsx_space_expr_text = format!("{{\"{}\"}}", " ".repeat(spaces_between_count));
+    let jsx_space_expr_text = format!("{{{}{}{}}}", get_quote_char(context), " ".repeat(spaces_between_count), get_quote_char(context));
```

Comment:
Was tempted to pull this out into a `fn` but it wasn't before so I left it. Happy to pull it out should you want to.

Reply:
That's ok! It's a minor thing. I'm going to do this after merge to expedite a release.